server/index.js

```diff
@@ -0,0 +1,22 @@
+import express from 'express';
+import swaggerUi from 'swagger-ui-express';
+import apiDocumentation from '../swagger.json';
+
+const app = express();
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+const basePath = '/api/v-unknown';
+
+app.listen(process.env.PORT || 3000, () => {
+  console.log('server is running on port 3000');
+});
+app.use(`${basePath}/documentation`, swaggerUi.serve, swaggerUi.setup(apiDocumentation));
+app.get('**', (req, res) => {
```

Comment:
I think you need to test this endpoint

Reply:
Yeah definitely, this was tested from the chore am working on called Setup unit testing and after pushing this chore I will be having testing environment. with one test of that endpoint.